src/utils/mail.py

```diff
@@ -30,19 +29,22 @@ def __init__(self, cfgFilePath):
             self._smtp_port = config.get("MAIL", 'port')
             self._email_pass = config.get("MAIL", 'password')
             self._send_from = config.get("MAIL", 'email')
-            self._to_emails = config.get("MAIL", 'toEmails').split(COMMA)
-            self._kindle_emails = config.get("MAIL", 'kindleEmails').split(COMMA)
+            self._to_emails = list(filter(None, config.get("MAIL", 'toEmails').split(COMMA)))
+            self._kindle_emails = list(filter(None, config.get("MAIL", 'kindleEmails').split(COMMA)))

```

Comment:
there is a try except clause in _send_email (mail.py) which will handle this case, but yeah, i 'll add it